README.md

```diff
@@ -108,29 +108,32 @@ Results using Ruby 3.0.2:
 
 |Method arguments|`Dry::Core` (0.7.1)|`Memery` (1.4.0)|
 |--|--|--|
-|`()` (none)|1.57x|19.86x|
-|`(a)`|2.49x|12.09x|
-|`(a, b)`|1.10x|5.24x|
-|`(a:)`|2.27x|24.09x|
-|`(a:, b:)`|1.09x|10.65x|
-|`(a, b:)`|1.05x|10.27x|
-|`(a, *args)`|2.10x|5.01x|
-|`(a:, **kwargs)`|1.61x|6.24x|
-|`(a, *args, b:, **kwargs)`|1.05x|2.94x|
+|`()` (none)|1.51x|19.82x|
+|`(a)`|2.30x|11.38x|
+|`(a, b)`|0.45x|2.10x|
+|`(a:)`|2.20x|22.83x|
+|`(a:, b:)`|0.49x|4.53x|
+|`(a, b:)`|0.46x|4.35x|
+|`(a, *args)`|0.89x|2.03x|
+|`(a:, **kwargs)`|0.82x|3.18x|
+|`(a, *args, b:, **kwargs)`|0.60x|1.62x|
+
+\* `Dry::Core` is faster for some cases, but at the risk of
```

Comment:
@jemmaissroff @ms-ati thoughts on this language?

Reply:
I don't love framing this as a direct "They're faster" -- I wonder if we note it without mentioning speed?